certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -4880,6 +4880,148 @@
         }
       }
     },
+    "/v2/FlowV2/ClassifyDocument": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "description": "Classify or Identify documents based on the file content. Please refer here for more info - https://dev.pdf4me.com/power-automate/documentation/actions/classify-document/",
+        "operationId": "ClassifyDocument_V1",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "required": false,
+            "schema": {
+              "properties": {
+                "docContent": {
+                  "description": "The content of the input file",
+                  "format": "byte",
+                  "title": "File Content",
+                  "type": "string",
+                  "x-ms-visibility": "important"
+                },
+                "document": {
+                  "description": "document",
+                  "properties": {
+                    "Name": {
+                      "description": "Name of the file",
+                      "title": "File Name",
+                      "type": "string",
+                      "x-ms-visibility": "important"
+                    }
+                  },
+                  "type": "object"
+                }
+              },
+              "required": [
+                "docContent"
+              ],
+              "type": "object"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Completed successfully!",
+            "schema": {
+              "properties": {
+                "parseId": {
+                  "description": "ParseId",
+                  "title": "Parse Id",
+                  "type": "string",
+                  "x-ms-visibility": "important"
+                },
+                "templateName": {
+                  "description": "Template Name",
+                  "title": "Template Name",
+                  "type": "string",
+                  "x-ms-visibility": "important"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "summary": "Classify Document",
+        "x-ms-api-annotation": {
+          "family": "ClassifyDocument",
+          "revision": 1,
+          "status": "Preview"
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/v2/FlowV2/ParseDocument": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "description": "Parse or read content from PDF based on preconfigured templates. Please refer here for more info - https://dev.pdf4me.com/power-automate/documentation/actions/parse-document/",
+        "operationId": "ParseDocument_V1",
```

Comment:
@varunynoox operationIds should be in Camel Case. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

Reply:
operaitonid has been renamed